src/components/interactive/ButtonGroup.jsx

```diff
@@ -5,20 +5,41 @@ import combine from '../../util/combine';
 
 const ButtonGroup = ({ buttons, onChange, ...restBtnGroup }) => (
   <div {...restBtnGroup} className="btngroup">
-    {buttons.map(({ value, content, key, Icon, onClick, label, ...restBtn }) => (
-      <GroupButton
-        key={key || label}
-        onClick={onChange ? combine(() => onChange(label || value), onClick) : onClick}
-        label={label}
-        Icon={Icon}
-        {...restBtn}
-      />
-    ))}
+    {/* `isFirstButton` and `isLastButton` are removed to guard against them being passed in. */}
```

Comment:
I don't know if this is really necessary to explicitly called out. There's a bunch of other stuff here that's also been removed so that it's not passed into rest props.

Reply:
I called this out because they're removed _and_ not passed in or used otherwise. The others are. Just wanted to be clear that they weren't accidentally left off.

Reply:
Talked through offline. Makes perfect sense to me. We would never want the consumer to be able to define isFirstButton on any ol random button 👍